.github/workflows/nodejs.yml

```diff
@@ -15,7 +15,7 @@ jobs:
 
     strategy:
       matrix:
-        os: [macOS-latest, ubuntu-latest]
+        os: [macOS-latest]
```

Comment:
Our build process needs some more work for each platform, hence for this PR, I only concentrate on macOS